### DISCUSSION TITLE
Metallicities over time/redshift - a community project?

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#213] [Astrophysics and Physics]
_Background_: the standard model says that "early" galaxies should be metal-poor. Alternative models need to show metallicity isn't correlated to redshift.

----

I made this page to show measurements of metallicity for local and "early" objects. 

https://mikehelland.github.io/hubbles-law/other/metallicity.htm

![image](https://github.com/a-cosmology-group/acg/assets/19311107/d8ff1c42-1e5f-468d-bd52-22547a40c614)

----

**Update, May 10, 2024**

I've been adding data and WOW, look at DESI in fuchsia!

![image](https://github.com/a-cosmology-group/acg/assets/19311107/9d74ba19-b8f5-40fc-984c-3ba850bc0759)

I've also updated the tool to interactively turn on and off datasets.

----

The measurements I've been collecting are $12 + log_{10}(O/H)$. I put them in a JSON file here, with their references, which is what builds the graph, table, and refs on the page:

https://github.com/mikehelland/hubbles-law/blob/master/data/metallicity.json

### A community project?

If anyone has other measurements to add to this, I think you can just fork my project, make a change to metallicity, and do a pull request. Maybe we can make a compelling case against the metallicity-z relationship.

**Comment by Francois-Zinserling** [comment]
This is very interesting. I don't have much to add, but have some questions and maybe stimulate conversation?

 - What I read is that distant (high redshift) galaxies, are presumed to be young stars/galaxies at time of emission, having low metallicity because metals take time to produce from Hydrogen. This is not observed, but results show that 'young' galaxies are more metallic that they should have been, and an almost absence of very low metallicity? 
 - 12+log(O/H) is a measure of the evolution of Oxygen into predominantly Hydrogen stars. Clearly log(O/H) is negative to get the results from your chart, and as one would expect because there is more hydrogen - almost always. The smaller O is, the bigger the subtraction. Then again from your chart it appears if there are a lot of low O stars/galaxies nearby, but erm ... almost none in the distance. 

Am I at least understanding it right so far? Apologies - cosmology is not my strong field, despite my presence here in this group. :)

**Reply by mikehelland** [reply]
Sounds about right to me. I added another source that I found, it's the orange boxes:

![image](https://github.com/a-cosmology-group/acg/assets/19311107/ac1ada10-8b0c-4214-84c7-5ab3211dd4fd)

So this does show more of what the mainstream predicts, as far as the earliest galaxies being lower in the metals. But, as you point out, the extremely metal poor galaxies are very near.

One thing to add is this:

>The most reliable metallicity measurements are based on the flux ratio of the [O iii] λ4363 line against [O iii] λ5007. The technique is called the Te method, because it determines the electron temperature (Te) of the gas, and hence the gas-phase oxygen-to-hydrogen (O/H) abundance (Aller [1984](https://iopscience.iop.org/article/10.3847/0004-637X/828/2/67#apjaa2f42bib2); Izotov et al. [2006](https://iopscience.iop.org/article/10.3847/0004-637X/828/2/67#apjaa2f42bib41)). However, detecting [O iii] λ4363 is difficult, because it is weak and almost undetectable in metal-rich galaxies.

https://iopscience.iop.org/article/10.3847/0004-637X/828/2/67

Measuring metallicity this way (O/H) seems to preclude metal rich galaxies. Apparently the line we use to detect O gets "washed" out if there is a lot of metal.

**Reply by RedshiftDrift** [reply]
Other ratios of spectroscopic lines are also related to metallicity, such as [Fe/H].

According to LCDM, young galaxies should mostly be found early (large z) in the universe, but galaxies are still being born today confirmed by low-metallicity galaxies at low redshift.

According to LCDM, old galaxies should mostly be found at low redshift.  But since they have been finding old galaxies at high redshift, galaxy formation theoriest are struggling to find processes that could explain why mature-looking, high-metallicity galaxies.  A large number of 'tweaks' are added to galaxy formation models.  Some are:
1) an accretion rate larger than the "Chandrasekhar accretion rate", meaning that stars collect more gas than the quantity available to them,
2) a large population of black holes that improve the efficiency of star formation beyond what's normally possible,
3) other miracles of that type.

As usual, cosmologists don't see any problem with LCDM, they claim we need better 'galaxy formation' models.

**Reply by RedshiftDrift** [reply]
@mikehelland this is a nice graph showing **no trend** as a function of time.  The hard part is to remove selection biases (e.g. high [O/H] galaxies might have an overall spectral colour that makes then harder to find at specific redshifts, with specific telescopes like Hubble...)

It is difficult for the non-expert to find data resources.  You may write to the author of each paper and ask if they know about such a compilation, or if they know of a review paper that would list what is known so far.

**Comment by mikehelland** [comment]
Here's a video about metallicity that includes the graphs and data I found:

https://www.youtube.com/watch?v=RKso3RakFrU

Here's a "short" that basically does a montage kind of thing a measurements from 2004 to 2024:

https://www.youtube.com/shorts/6bIFuO13udY

**Comment by LifeIsStrange** [comment]
@mikehelland Thanks a lot for making this compilation, clarifying the metallicity over redshift relation (and related stellar mass/redshift, stellar formation rate (SFR)/redshift, and the hybrid permutations (mass/metallicity correlation, SFR/mass, SFR/metallicity, luminosity/others), is a big open problem and I am not aware of a complete up to date compilation of the disparate datasets. It is however known that there is a tension between them, a recent paper showed that the variance, while it can be explained by different instruments, can actually largelly be explained by different metallicities tracers/analytical methods. One could still hope that modern large surveys like DESI would be more statistically robust even though there are still systematics.
As a reminder the big bang theory by design stipulate, counter intuitivelly, that the stellar formation rate should increase over time but then at high redshift should reach an inflection point and decrease because while the universe, in the theory, was energetically denser, the eddington accretion speed requires the gases to "cool" IIRC to catalyze accretion. Of course there are many possible additional astrophysical mechanisms (dust obscuration, MOND, etc) that can alter accretion speed/stability and also observational biases such as the fundamental https://en.wikipedia.org/wiki/Malmquist_bias .

Anyway if I understand correctly, the fuschia dots you added are from the DESI early data release, and while it might seems big, it is order of magnitudes smaller than the already released DESI 2024 release
cf:
https://data.desi.lbl.gov/doc/papers/
While they released papers of analyzing the DESI 2024 data (no analysis of metallicity though despite spectrometric data) it seems that they haven't released the data yet for some mysterious reason
> Data Release 1 covering the first year of DESI main survey data (through June 2022) will be publicly released in 2025. Papers with cosmology results using DR1 data are available on the [papers page](https://data.desi.lbl.gov/doc/papers).

So we have papers analyzing the DR1 (not early data release) but we can't access said data until a year latter? Doesn't make any sense to me but hey not my area of expertise.

Besides DESI DR1 and JWST JADES, and the old but large SDSS/BOSS, the largest spectroscopic sky survey will be from euclid, in december.
However they recently have released some papers in a small data release but I'm not sure wether they contain useful metallicity data.
Coordinated Release "Euclid on Sky," May 2024
https://www.euclid-ec.org/science/publications/
mass function:
https://arxiv.org/abs/2405.13501
this one talk a bit about metallicity
https://arxiv.org/pdf/2405.13503

bonus: the obscolete madau plot
https://ned.ipac.caltech.edu/level5/Illingworth/Ill5.html

**Reply by mikehelland** [reply]
I think it's just how things go. The ones performing the observations focus on that. They publish the objective data. That goes to researchers that can do things model and theory dependent like measure O/H or whatever their area of expertise.

I know that JWST data for many projects gets embargoed to the team that proposed the experiment get time to analyze before it goes public domain. Something like that could be what's involved here. Hey, we waited so long for JWST. Things seem to be happening quickly compared to that.

**Comment by LifeIsStrange** [comment]
https://arxiv.org/abs/2407.17110

**Reply by LifeIsStrange** [reply]
might be relevant https://arxiv.org/abs/2409.06174

**Reply by LifeIsStrange** [reply]
nice paper on evolution https://arxiv.org/abs/2502.04702

**Reply by LifeIsStrange** [reply]
Contradiction to "might be relevant paper"
"four elements to rule them all"
https://arxiv.org/abs/2504.18532

**Comment by LifeIsStrange** [comment]
Record breaking low metallicities
https://arxiv.org/abs/2505.03873

**Comment by LifeIsStrange** [comment]
https://arxiv.org/abs/2508.18369
This work provides the most detailed view of MZR and FMR evolution from the present day through Cosmic Noon with a fine time sampling of  Gyr, setting a robust baseline for metallicity evolution studies at  with JWST.

Also @mikehelland iirc DESI DR2 data has been released

**Comment by Francois-Zinserling** [comment]
Not sure if you had listed this one before but I recently came across this article:
[The Most Metal-Free Star Ever Observed is Found In Our Own Backyard](https://www.universetoday.com/articles/the-most-metal-free-star-ever-observed-is-found-in-our-own-backyard)

What I find intriguing though ...

> The star is known as SDSS J0715-7334. It is a red giant star in the halo of the Large Magellanic Cloud.

, is that it is right on our doorstep. (backyard as they quote), meaning that signal was sent 'yesterday'. 
Now, given the BB dialogue, I would have expected to see this kind of signature in a far-off (long time ago) signal that was sent from a star ~10b yo. By that argument there should be no region of space in our (time and) vicinity remaining that has enough dust to create a new star that is not already heavily metal-loaded.
But it is here, and it is likely not that old. (my opinion) The argument is that it is a low mass star and thus has longevity, but stars don't live in isolation, as even a supernova from a star next door will pepper it with huge amounts of metals, and being in an orbit for that long will lead to accruing much dust from the environment. 

Maybe they have found a regenerator. I can dream.

**Reply by LifeIsStrange** [reply]
A dream that I wish too :)
